superdesk/publish/subscribers.py

```diff
@@ -103,12 +103,14 @@ class SubscribersService(CacheableService):
     cache_lookup = {"is_active": True}
 
     def get(self, req, lookup):
+        fields = ("secret_token", "password", "apiKey", "access_key_id", "secret_access_key")
```

Comment:
just wondering why it's set here and not in the function itself, or as a property on the class